reana_client/cli/files.py

```diff
@@ -67,10 +73,12 @@ def get_files(ctx, workflow, _filter,
     Examples: \n
     \t $ reana-client ls --workflow myanalysis.42
     """
+    JSON = 'json'
+    URL = 'url'
+    FILE_PATH = "%s/api/workflows/%s/workspace/%s"
```

Comment:
If you would be able to retrieve [this string](https://github.com/reanahub/reana-commons/blob/master/reana_commons/openapi_specifications/reana_server.json#L1835) from the `api_client` you could do as follows (and we would be save from having this string outdated in the future 😄):

```python
In[1]: "/api/workflows/{workflow_id_or_name}/workspace/{file_name}".format(workflow_id_or_name='1234', file_name='test.json')                                                     
Out[2]: '/api/workflows/1234/workspace/test.json'
```